data-models/common/climate/schema.json

```
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://akenza.github.io/device-types-library/data-models/common/climate/schema.json",
+  "$defs": {
+    "temperatureC": {
+      "measurementType": "common/climate/temperatureC/C",
```

```suggestion
      "measurementType": "common/climate/temperatureCelsius/C",
```

As already proposed in the name.